tests/10-issue.js

```diff
@@ -64,206 +69,248 @@ describe('Issuers - BitstringStatusList',
           async function() {
             this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20statusListIndex%20property%20MUST%20be%20an%20arbitrary%20size%20integer%20greater%20than%20or%20equal%20to%200%2C%20expressed%20as%20a%20string%20in%20base%2010';
-            issuedVc.credentialStatus.should.have.own.property(
-              'statusListIndex').to.be.a('string',
-              'Expected statusListIndex to be a string.'
-            );
-            assert(
-              String(parseInt(issuedVc.credentialStatus.statusListIndex)) ===
-              issuedVc.credentialStatus.statusListIndex,
-              'Expected statusSize value to be an integer ' +
-              'expressed as a string in base 10.'
-            );
-            parseInt(issuedVc.credentialStatus.statusListIndex).should.be.gte(0,
-              'Expected credentialStatus.statusListIndex to be >= 0.');
+            for(statusEntry in statusEntries) {
+              statusEntries[statusEntry].should.have.own.property(
+                'statusListIndex').to.be.a('string',
+                'Expected statusListIndex to be a string.'
+              );
+              assert(
+                String(parseInt(statusEntries[statusEntry].statusListIndex)) ===
+                statusEntries[statusEntry].statusListIndex,
+                'Expected statusSize value to be an integer ' +
+                'expressed as a string in base 10.'
+              );
+              parseInt(statusEntries[statusEntry].statusListIndex).
+                should.be.gte(0,
+                  'Expected credentialStatus.statusListIndex to be >= 0.');
+            }
           });
           it('The statusListCredential property MUST be a URL to a ' +
             'verifiable credential.',
           async function() {
             this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20statusListCredential%20property%20MUST%20be%20a%20URL%20to%20a%20verifiable%20credential.';
             // TODO check for URI
-            issuedVc.credentialStatus.should.have.own.
-              property('statusListCredential').to.be.
-              a('string',
-                'Expected credentialStatus.statusListCredential to be a URL.'
-              );
-            const {slc} = await getSlc({issuedVc});
-            should.exist(slc,
-              'Expected statusListCredential to resolve to a ' +
-              'Verifiable Credential');
-            testSlCredential({slCredential: slc});
+            for(statusEntry in statusEntries) {
+              statusEntries[statusEntry].should.have.own.
+                property('statusListCredential').to.be.
+                a('string',
+                  'Expected credentialStatus.statusListCredential to be a URL.'
+                );
+              const {slc} = await getSlc(statusEntries[statusEntry]);
+              console.log(statusEntries[statusEntry]);
```

Comment:
is the console.log intentionally left in?

Reply:
Good catch, this was from me running tests. It's been removed